Triplehelix490/sketch_jul12a_fade/sketch_jul12a_fade.ino

```diff
@@ -0,0 +1,24 @@
+
+int pin = 9;
+int brightness = 0;
+int fadeAmount = 255;
+int timeOn = 4000;
+int timeOff = 2000;
+
+//setup code: 
+void setup() {
+    pinMode(pin, OUTPUT);
+}
+  
+//main code
+
+void loop (){
+
+    analogWrite(pin, HIGH);
+    brightness = brightness + fadeAmount
+      delay(timeOn);
+       analogWrite(pin, LOW);
+          delay(timeOff);
+
+
+}
```

Comment:
Well Done Adam, I liked the way you have structured the Arduino coding, its very cool, Keep up this great work and you will smash this Adam. #kingAdam# #Never Give Up#